schema/CVE_Record_Format.json

```diff
@@ -301,10 +306,33 @@
                             {
                                 "required": ["version", "status", "versionType", "lessThan"]
                             },
+                            {
+                                "required": ["status", "versionType", "lessThan"]
+                            },
                             {
                                 "required": ["version", "status", "versionType", "lessThanOrEqual"]
-                            }
+                            },
+                            {
+                                "required": ["status", "versionType", "greaterThan"]
+                            },
+                            {
+                                "required": ["status", "versionType", "greaterThanOrEqual"]
+                            },
                         ],
+                        "if": {
+                            "properties": {
+                                "versionType": {"const": "semver-2.0.0"}
+                            }
+                        },
+                        "then": {
+                            "properties": {
+                                "version": { "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"},
+                                "lessThan": { "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"},
+                                "lessThanOrEqual": { "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"},
+                                "greaterThan": { "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"},
+                                "greaterThanOrEqual": { "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"},
```

Comment:
Shouldn't these use the `semver-2.0.0-version` definition?

Reply:
They should. Good catch and updated in https://github.com/CVEProject/cve-schema/pull/371/commits/1e9111771d9df1d2c4a45863e7b6e851be6ff2b3